src/applications/gqrx/file_resources.cpp

```diff
@@ -44,7 +44,8 @@ std::string receiver::get_random_file(void)
                 QDataStream stream(&temp_file);
                 for (size_t i = 0; i < 1024*8; i++) stream << qint8(rand());
             }
-            std::cout << "Created random file " << path << std::endl;
+            temp_file.close();
+            std::cout << "Created random file " << path << std::endl;            
```

Comment:
Some white space was accidentally added to this line.
```suggestion
            std::cout << "Created random file " << path << std::endl;
```

Reply:
Removed AFAICS. I *do* have a proper editor.

Reply:
Proper editor or not, line 48 still has 12 trailing spaces added to it in this commit. :smile:

Reply:
Oh, I blame the editor at Github. Fixed now

Reply:
Looks good now. :+1: